.gitignore

```diff
@@ -0,0 +1,2 @@
+LICENSE
+README.md
```

Comment:
These files should not be ignored.

Reply:
Thank you for pointing this out! Will update.